manpages/lumina-screenshot.8

```diff
@@ -1,24 +1,29 @@
-.Dd October 27, 2017
+.Dd October 31, 2017
 .Dt LUMINA-SCREENSHOT 8
 .Os 1.3.3
 
 .Sh NAME
 .Nm lumina-screenshot
-.Nd This is a utility for taking screenshots of the local desktop session and saving
-them on the system.
+.Nd a utility for taking screenshots of the local desktop
+session and saving them on the system.
 
 .Sh SYNOPSIS
 .Nm
 
 .Sh DESCRIPTION
 .Nm
-Takes screenshots of connected monitors. Then, it can view, crop, zoom, and
-save the screenshot. There is also an option to open the screenshot with a different application.
-.Nm
-has an option to copy, allowing the user to easily copy and paste the screenshot. Under
-the settings tab, the screenshot tool has options to capture the entire session, a single screen, a
-single window, or a selected area. By default, the utility is bound to the "PrintScreen" keyboard
-button.
+takes screenshots of connected monitors.
+Then, it can view, crop, zoom, and save the screenshot.
+There is also an option to open the screenshot with a different
+application.
+Under the settings tab, the screenshot tool has options to capture the
+entire session, a single screen, a single window, or a selected area.
+By default, the utility is bound to the "PrintScreen" keyboard button.
+
```

Comment:
We should describe the functionality of the utility without confusing the reader as to what they can do from the CLI.